tests/data/either_example.rs

```diff
@@ -0,0 +1,11 @@
+use extendr_api::prelude::*;
+
+#[extendr(use_try_from = true)]
+fn matrix_sum(input : ArrayView2<Rfloat>) -> Rfloat {
```

Comment:
You have an `either`-example that doesn't use `either`? 
Lol.. I..... sort of understand why. But maybe we should have a patch.crates-io for `extendr`?
The published version wouldn't really contain and use this example anyways.
You can annotate it nicely?

Reply:
Right, you can add a `[patch]` section ot the `.cargo/config.toml` or add it via... commandline option..
Thus we "just" need to ignore `.cargo/config.toml` file in the `.Rbuildignore`-file.

Reply:
Okay. This is a little bit of a headache, because do you maintain `rextendr` so it works with `extendr` published, or `extendr` develop? If you choose the former, than newer features will stagnate and maybe we'll have issues. If you choose the latter then we have to decide to publish `libR-sys`, `extendr` and `rextendr` simultanously.

Reply:
A little bit overwhelming, but I'll try to respond.
1. It is a mistake. I originally tried using `either` but then remembered it has not been released. I changed code but no the name of the file
2. I track only dependencies. If someone hacks into `patch` or whatever, it is up to that person to define features and imports. My job is to make it `rust_function("fn ndarray_fn() {}", features = "ndarray")`
3. I chose to limit features to a subset defined in the released `extendr`. It maybe unwise. I think I will rework it to not be limited to a fixed set of features, but if an unrecognized feature is passed, I will raise a warning. Thanks for noticing!